GitUI/CommandsDialogs/SettingsDialog/Pages/DiffViewerSettingsPage.cs

```diff
@@ -1,14 +1,26 @@
 ﻿using GitCommands;
+using ResourceManager;
 
 namespace GitUI.CommandsDialogs.SettingsDialog.Pages
 {
     public partial class DiffViewerSettingsPage : SettingsPageWithHeader
     {
+        private readonly TranslationString _showDiffForAllParents = new TranslationString(@"If set, show all differences (not just the first differences).
+
+For a single selected commit, show the difference to the parent commit.
+For a single selected merge commit, show all parents.
```

Comment:
What does "_show all parents_" mean? Do we mean a difference between all parents of a merge commit?

Reply:
A merge commit was the initial use. Usage has been expanded.
A more descriptive would negate the setting and name it to "Show only one diff".